ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool. 
+
+## Introduction 
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema. 
```

Comment:
Also, an introduction in documentation needs to be more practical. Like what you could achieve if you use this specific integration. Readers here already have some understanding about what is YDB and might have already stumbled upon migration issues.

Reply:
the same applies for the rest of the text